cmd/hack.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+  "log"
+
+  "github.com/Originate/gt/cmd/config"
+  "github.com/Originate/gt/cmd/git"
+  "github.com/Originate/gt/cmd/script"
+  "github.com/Originate/gt/cmd/steps"
+  "github.com/Originate/gt/cmd/util"
+
+  "github.com/spf13/cobra"
+)
+
+var abortFlag bool
+var continueFlag bool
+
+var hackCmd = &cobra.Command{
+  Use:   "hack",
+  Short: "Create a new feature branch off the main development branch",
+  Long:  `Create a new feature branch off the main development branch`,
+  Run: func(cmd *cobra.Command, args []string) {
+    steps.Run(steps.RunOptions{
+      Command: "hack",
+      IsAbort: abortFlag,
+      IsContinue: continueFlag,
+      IsSkip: false,
+      SkipMessage: "",
+      StepListGenerator: func() steps.StepList {
```

Comment:
I don't believe thats possible as generating the steps requires checking the preconditions and we don't want to do that if we are aborting / continuing.